File: 백준/Bronze/23971. ZOAC 4/ZOAC 4.py
import math
# H행 W열에서 모든 자리에 책상을 비치하며 간격은 1칸씩 띄워서 사람들이 착석 가능

def solution(H,W,N,M):
  x = math.ceil(H/(N+1))
  y = math.ceil(W/(M+1))
  return x*y

H,W,N,M = list(map(int, input().split()))
print(solution(H,W,N,M))

File: 백준/Bronze/23971. ZOAC 4/ZOAC 4.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        String[] S = br.readLine().split(" ");
        int H = Integer.parseInt(S[0]);
        int W = Integer.parseInt(S[1]);
        int N = Integer.parseInt(S[2]);
        int M = Integer.parseInt(S[3]);

        System.out.println(Solution(H, W, N, M));
    }

    
    
    
    
    public static int Solution(int H, int W, int N, int M) {
        int x = (H + N) / (N + 1);
        int y = (W + M) / (M + 1);
        return x * y;
    }

    
    
    
    
}